FakebookNotifications/FakebookNotifications.WebApi/Startup.cs

```diff
@@ -34,6 +34,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<INotificationsContext, NotificationsContext>();
             services.AddScoped<IUserRepo, UserRepo>();
             services.AddScoped<INotificationsRepo, NotificationsRepo>();
+            
```

Comment:
unnecessary diff

Reply:
previous review comment unaddressed.

Reply:
I believe we were trying to add some things, but then deleted the changes so that one extra line stayed. I guess the right call would have been to restore the file.